### DISCUSSION TITLE
后端代码中的editDomain和事务（比如InternalTransaction 这个已经是EMF框架中的了）是干什么的？

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#138] [ModelServer]
请教一下，后端代码中的editDomain和事务（比如InternalTransaction 这个已经是EMF框架中的了）是干什么的？谢谢。

**Comment by eneufeld** [comment]
I assume you are talking about the modelserver. The modelserver is based on EMF. Please check this tutorial: https://eclipsesource.com/blogs/tutorials/emf-tutorial/ 
Otherwise please specify your question.

Please also post your questions in english as you will help others to find your question and possible answers to it. Thank you.
